capstone/src/main/java/com/google/sps/search/CsvReader.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.search;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CsvReader {
+
+  public CsvReader() {}
+
+  public List<String> parseCorpus(String file) {
+    List<String> names = new ArrayList<String>();
+    Scanner in = null;
+    try {
+      in = new Scanner(new File(file));
+    } catch (FileNotFoundException e) {
+      System.err.println("File does not exist.");
+      return null;
+    }
```

Comment:
You should make sure to close your file in the catch otherwise you may have unexpected consequences